unity/CAVE/Assets/W3D_Translator/Classes.cs

```diff
@@ -217,25 +237,30 @@ public enum VerticalAlignments {
         [XmlAttribute(AttributeName="depth")]
         public float Depth;
 
-        public void GenerateTMP(GameObject gameObject, Color color) {
-            TextMeshPro tmp = gameObject.AddComponent<TextMeshPro>();
-            
-            // Change TMP Defaults
-            tmp.autoSizeTextContainer = true;
-            // TODO (64): Validate default values (font size, wrapping, overflow, etc)
-            tmp.fontSize = 10;
-            tmp.enableWordWrapping = false;
-            tmp.overflowMode = TextOverflowModes.Truncate;
+        public GameObject GenerateTMP(bool isLink, Color color) {
+            // Instantiate TextMeshPro or TextMeshProUGUI prefab
+            // TODO (64): Validate prefab settings
+            GameObject gameObject = UnityEngine.Object.Instantiate(
+                Resources.Load<GameObject>("Prefabs/tmp" + (isLink ? "GUI" : ""))
+            );
+            TMP_Text tmp = gameObject.GetComponent<TMP_Text>();
```

Comment:
maybe a better name here? When I see `tmp` I think temporary object, but this is the actual text object that's used in the scene right?

Reply:
Yeah, I do suppose I only ever use TextMeshPro for text in Unity (the old way is all but deprecated) so it's safe to just call it `text` or something like that